test/core/rebalancingSetToken.spec.ts

```diff
@@ -367,48 +449,121 @@ contract('RebalancingSetToken', accounts => {
     });
   });
 
-  describe('#burn', async () => {
-    let subjectBurner: Address;
+  describe('#burn: Called on Rebalancing Token', async () => {
+    let subjectIssuer: Address;
     let subjectQuantity: BigNumber;
     let subjectCaller: Address;
 
-    let newRebalancingToken: Address;
-
     beforeEach(async () => {
+      const naturalUnit: BigNumber = ether(2);
       components = await erc20Wrapper.deployTokensAsync(2, deployerAccount);
-      factory = await coreWrapper.deploySetTokenFactoryAsync(coreAccount);
+      await erc20Wrapper.approveTransfersAsync(components, transferProxy.address);
+
+      const currentComponentAddresses = _.map(components, token => token.address);
```

Comment:
Could we put the map function in a util or something? It's a bit jarring to see this low level stuff in a test

Reply:
Those tests are gonna be deleted anyway but I also don't think it's worthwhile to create a full other util for one line of code.

Reply:
Sorry not gonna be deleted was looking at wrong test suite but still don't think its worth creating its own util... _.map is the util.